Library/Homebrew/utils/user.rb

```diff
@@ -6,6 +6,21 @@
 require "system_command"
 
 class User < DelegateClass(String)
+  def self.automation_access?
+    *_, status = system_command "osascript", args: [
+      "-e", "with timeout of 0.1 seconds",
```

Comment:
Has this timeout been reliable in your tests? Reason I ask: https://github.com/Homebrew/homebrew-cask/issues/19076#issuecomment-421836262

Reply:
In my tests, even `0.00001 seconds` was reliable.

Reply:
Increased it to 0.5 since it seems to take a tad longer the first time after a full reboot.